### ISSUE TITLE
Crash report - runtime error: index out of range

### ISSUE THREAD

**Issue by gildo** [github/hub#1296]
Error (runtime.errorString): `runtime error: index out of range`

Stack:

```
goroutine 1 [running]:
panic(0x77e480, 0xc420010130)
    /usr/lib/go/src/runtime/panic.go:458 +0x243
github.com/github/hub/git.WorkdirName(0xc4200cf640, 0x10, 0x0, 0x0)
    /tmp/go/src/github.com/github/hub/git/git.go:70 +0xb6
github.com/github/hub/commands.create(0x99c620, 0xc420096090)
    /tmp/go/src/github.com/github/hub/commands/create.go:66 +0x6a
github.com/github/hub/commands.(*Command).Call(0x99c620, 0xc420096090, 0x40b6b9, 0x7f0248)
    /tmp/go/src/github.com/github/hub/commands/commands.go:47 +0x91
github.com/github/hub/commands.(*Runner).Call(0xc4200116e0, 0x99c620, 0xc420096090, 0x6, 0xc420073f00, 0x450310)
    /tmp/go/src/github.com/github/hub/commands/runner.go:93 +0x3f
github.com/github/hub/commands.(*Runner).Execute(0xc4200116e0, 0x811620, 0xc420013650, 0xc4200001a0)
    /tmp/go/src/github.com/github/hub/commands/runner.go:85 +0x2cb
main.main()
    /build/hub/src/github-hub-7988c4b/main.go:12 +0x49

```

Runtime:

```
GOOS: linux
GOARCH: amd64
```

Version:

```
git version 2.10.0
hub version 2.2.5
```

**Comment by mislav** [comment]
Hi @gildo, this was fixed in v2.2.6. Could you try updating to latest version v2.2.9? Thanks!

**Comment by gildo** [comment]
@mislav thanks. I've flagged the package out of date on the official Archlinux Report.

**Comment by mislav** [comment]
Hmm, now that I look at the crash trace, I'm not sure if this particular bug was fixed. I was thinking of another crash that got fixed in v2.2.6. Is this reproducible every time? Is it just a particular repository/directory location, or everywhere? You can use `hub --noop create` to safely run the command anywhere without making actual changes on GitHub. The `--noop` flag just prints what would be done without actually posting any data.

**Comment by mislav** [comment]
Closing as duplicate of https://github.com/github/hub/issues/1331
